src/sentry/api/endpoints/auth_index.py

```diff
@@ -100,8 +121,17 @@ def put(self, request: Request):
             return Response(status=status.HTTP_401_UNAUTHORIZED)
 
         validator = AuthVerifyValidator(data=request.data)
-        if not validator.is_valid():
-            return self.respond(validator.errors, status=status.HTTP_400_BAD_REQUEST)
+
+        from_superuser_modal = request.data.get("isSuperuserModal")
```

Comment:
should you add `isSuperUserModal` to the `AuthVerifyValidator` and then access it via that?